docs/additional-deployment-options/cmr_launchpad_authentication.md

```diff
@@ -18,16 +18,16 @@ Steps for setting up the CMR Client to use Launchpad authentication can be found
 
     - Upload the PKI certificate `.pfx` file to S3
     - Use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/launchpad.pfx` as key
-      - `{{system_bucket}}` and `{{prefix}}` are configured in `app/config.yml`.
-      - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad` configuration in `app/config.yml`.
+      - `{{system_bucket}}` and `{{prefix}}` are configured in `cumulus-tf/terraform.tfvars`.
```

Comment:
```suggestion
      - `system_bucket` and `<prefix>` are configured in `cumulus-tf/terraform.tfvars`.
```

Reply:
Kind of nitpicky, but the `{{ }}` syntax is a leftover from Kes, so I'd rather not keep it to avoid confusion